src/interfaces/token.ts

```diff
@@ -63,22 +63,23 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
     const vaultBalances = await this.yearn.vaults
       .balances(address)
       .then(balances => balances.filter(token => token.balance !== "0"));
-
-    if (this.chainId === 1 || this.chainId === 1337) {
-      let zapperBalances = await this.yearn.services.zapper.balances(address);
-      const vaultBalanceAddresses = new Set(vaultBalances.map(balance => balance.address));
-      zapperBalances = zapperBalances.filter(balance => !vaultBalanceAddresses.has(balance.address));
-      return zapperBalances.concat(vaultBalances);
+    switch(this.chainId){
+      case 1:
+      case 1337:
+        let zapperBalances = await this.yearn.services.zapper.balances(address);
+        const vaultBalanceAddresses = new Set(vaultBalances.map(balance => balance.address));
```

Comment:
need to move this line to before this to be above the `switch` statement, and also remove the similar line from the block below

Reply:
and also run `yarn lint` after this fix please